### ISSUE TITLE
Dota Plus Assistant not working

### ISSUE THREAD

**Issue by eric1294** [ValveSoftware/Dota2-Gameplay#11186]
### Description

ONCE again, your **glorious** dota plus assistant fails to suggest builds anymore.

**Comment by Aphotic** [comment]
yeah, it been months/years that it has been faulty and very outdated.

**Comment by eric1294** [comment]
I'm gonna tell you right now that I'm getting really sick of this happening.

**Comment by Orenix** [comment]
![2023-08-30 23_46_04-Greenshot](https://github.com/ValveSoftware/Dota2-Gameplay/assets/51746598/2da3aaff-013c-4490-a7df-9ad863c5c66f)

Dota plus offers me the same item 3 times... Which I obviously already have.

30/08/2023

**Comment by Josie-Xu86** [comment]
same problem,i have tried revalidate files,but it still not work,fxxk valve

**Comment by UberPwnage** [comment]
I am having a similar problem for the past couple months where is suggest ridiculous items that make no sense and it loses track of which items I've already purchased.  This feature is flat out broken.
